backend/src/test/resources/graphql-test/test-results-with-count-query.graphql

```diff
@@ -1,7 +1,4 @@
-query GetFacilityResultsWithCount(
-  $facilityId: ID!
-  $nameType: String = "long"
-) {
+query GetFacilityResultsWithCount($facilityId: ID, $nameType: String = "long") {
```

Comment:
Removing the requirement for `facilityId` here to test the organization-level fetch available in the `TestResultResolver.testResultPage`